d2l-button-behavior.html

```diff
@@ -23,6 +23,12 @@
 			type: Boolean,
 			reflectToAttribute: true
 		},
+		/**
+		 * Indicates the button opens a menu.
+		 */
+		ariaHaspopup: {
+			type: String
+		},
```

Comment:
Could you also include aria-expanded? It would be useful for dropdown buttons to indicate the expanded/not-expanded state

Reply:
Yep, will add. I wonder if the dropdown behaviour should be setting that? It doesn't appear to be currently.

Reply:
It probably should. I'll create a PR.